### PR TITLE
Add restriction for lifetime parameter when creating schain

### PR DESCRIPTION
**Pull request by yavrsky**

### CODE REVIEW

File: contracts/Schains.sol [hunk 1]
```diff
@@ -100,7 +100,14 @@ contract Schains is Permissions {
      */
     function addSchain(address from, uint deposit, bytes calldata data) external allow("SkaleManager") {
         SchainParameters memory schainParameters = _fallbackSchainParametersDataConverter(data);
-        
+        ConstantsHolder constantsHolder = ConstantsHolder(contractManager.getContract("ConstantsHolder"));
+        uint schainCreationTimeStamp = constantsHolder.schainCreationTimeStamp();
+        uint minSchainLifetime = constantsHolder.minimalSchainLifetime();
+        require(now > schainCreationTimeStamp, "It is not a time for creating Schain");
```
Comment (DimaStebaev) [inline contracts/Schains.sol]: ```>=```

File: contracts/Schains.sol [hunk 2]
```diff
@@ -100,7 +100,14 @@ contract Schains is Permissions {
...
+        require(
+            schainParameters.lifetime > minSchainLifetime,
```
Comment (DimaStebaev) [inline contracts/Schains.sol]: ```>=```

**Comment by DimaStebaev** [comment]
If write small test to cover changes in ConstantsHolder contract code coverage will be awesome!

**Review by payvint — APPROVED** [review]
Nice!

**Pull request merged**